Source/AudioPlayer.swift

```diff
@@ -153,6 +153,10 @@ public class AudioPlayer: NSObject {
             sound?.play()
         }
     }
+    
+    public func playWithDelay(duration: Int) {
+        Timer.scheduledTimer(timeInterval: TimeInterval(duration), target: self, selector: #selector(play), userInfo: nil, repeats: false)
```

Comment:
Instead of creating a new method, can you update the current `play` by adding an delay parameter: `play(withDelay delay: Int = 0)`?